src/posh-git.psm1

```diff
@@ -67,29 +68,39 @@ $GitPromptScriptBlock = {
         $prompt += Write-Prompt $promptPrefix
     }
 
-    # Write the abbreviated current path
+    # Get the current path - formatted correctly
     $promptPath = [PoshGitTextSpan]::new($settings.DefaultPromptPath)
     $promptPath.Text = $ExecutionContext.SessionState.InvokeCommand.ExpandString($promptPath.Text)
-    $prompt += Write-Prompt $promptPath
 
-    # Write the Git status summary information
-    $prompt += Write-VcsStatus
+    # Write the path and Git status summary information
+    if ($settings.DefaultPromptWriteStatusFirst) {
+        $prompt += Write-VcsStatus
+        $prompt += Write-Prompt $promptPath
+    }
+    else {
+        $prompt += Write-Prompt $promptPath
+        $prompt += Write-VcsStatus
+    }
 
     # If stopped in the debugger, the prompt needs to indicate that in some fashion
-    $hasInBreakpoint = [runspace]::DefaultRunspace.Debugger | Get-Member -Name InBreakpoint -MemberType property
-    $debugMode = (Test-Path Variable:/PSDebugContext) -or ($hasInBreakpoint -and [runspace]::DefaultRunspace.Debugger.InBreakpoint)
```

Comment:
We no longer have to check `Variable:/PSDebugContext` because >= v5 will have the `InBreakpoint` property.